nsv13/code/modules/research/designs/ship_weapon_designs.dm

```diff
@@ -200,3 +200,25 @@
 	build_path = /obj/item/ship_weapon/ammunition/railgun_ammo/uranium
 	category = list("Advanced Munitions")
 	departmental_flags = DEPARTMENTAL_FLAG_MUNITIONS
+
+
+//subspace cooling stuff
+/datum/design/board/subspace_cooler
+	name = "Machine Design (Experemental Subspace Cooling Unit)"
+	desc = "Allows for the construction of a Subspace Cooling Unit."
+	id = "subspace_cooler"
+	build_type = PROTOLATHE|IMPRINTER
+	materials = list(/datum/material/glass = 2000, /datum/material/copper = 2000, /datum/material/gold = 5000)
+	build_path = /obj/item/circuitboard/machine/cooling
```

Comment:
```suggestion
	build_path = /obj/item/circuitboard/machine/cooling/cooler
```
Otherwise this does nothing

Reply:
oops

Reply:
wait there's no cooler board, the base prefab for the board is the cooler board. this breaks